native/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -0,0 +1,257 @@
+
+            Path path = Paths.get(this.sourcePath, TARGET, SQL_PATH);
+
+            FileReader fileReader = new FileReader(path.toAbsolutePath().toString());
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            while ((sValue = bufferedReader.readLine()) != null) {
+                stringBuffer.append(sValue);
+            }
+            bufferedReader.close();
+            sqlLines = stringBuffer.toString().split(";");
+
+        } catch (BalException e) {
+            errStream.println(e.getMessage());
+            return;
+        } catch (IOException e) {
+            errStream.println("Error occurred while reading generated SQL scripts!");
+            return;
+        }
+        String url = String.format(JDBC_URL_WITHOUT_DATABASE, MYSQL,
+                configurations.get(HOST).replaceAll("\"", ""), configurations.get(PORT));
+        String user = configurations.get(USER).replaceAll("\"", "");
+        String password = configurations.get(PASSWORD).replaceAll("\"", "");
+        String database = configurations.get(DATABASE).replaceAll("\"", "");
+        Properties props = new Properties();
+        props.put("user", user);
+        props.put("password", password);
+        try {
+            connection = driver.connect(url, props);
+            ResultSet resultSet = connection.getMetaData().getCatalogs();
+            boolean databaseExists = false;
+            while (resultSet.next()) {
+
+                if (resultSet.getString(1).trim().equals(database)) {
+                    databaseExists = true;
+                    break;
+                }
+            }
+            if (!databaseExists) {
+                statement = connection.createStatement();
+                String query = String.format(CREATE_DATABASE_SQL, database);
+                statement.executeUpdate(query);
+                stdStream.println("Creating Database : " + database);
+            }
+            resultSet.close();
+            connection.close();
+            String databaseUrl = String.format(JDBC_URL_WITH_DATABASE, MYSQL,
+                    configurations.get(HOST).replaceAll("\"", ""), configurations.get(PORT),
+                    configurations.get(DATABASE).replaceAll("\"", ""));
+            if (!isTest) {
+                connection = driver.connect(databaseUrl, props);
+            } else {
+                connection = DriverManager.getConnection(databaseUrl, user, password);
+            }
+            statement = connection.createStatement();
+
+            for (String sqlLine : sqlLines) {
+                if (!sqlLine.trim().equals("")) {
+                    statement.executeUpdate(sqlLine);
+                }
+            }
+            statement.close();
+            connection.close();
+        } catch (SQLException e) {
+            errStream.println("Error occurred when creating database tables: " + e.getMessage());
+        }
```

Comment:
Shall we split the logic into different functions based on the tasks, like `readConfigToml`, `setupDBConnection` etc